### PR TITLE
CH-259: Data Store foundation.

### PR DESCRIPTION
**Pull request by akash2144**
Signed-off-by: Akash Dudhane <akash.a.dudhane@seagate.com>

# Problem Statement
- Create foundation for Alert Data Store : [CH-259](https://jts.seagate.com/browse/CH-259)

# Design
-  https://seagate-systems.atlassian.net/wiki/spaces/CH/pages/1074757633/Event+workflow+and+management+backend+LLD+WIP#Data-Stores

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [x] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

# Testing
```
[root@cortx-ha-headless-svc cortx-halo]# python3 test/py/main.py -m 'unit'
==================================================== test session starts ====================================================
platform linux -- Python 3.6.8, pytest-7.0.1, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /cortx-halo, configfile: pytest.ini
collected 8 items

common/test/test_alert_data_store.py::test_save_data PASSED                                                           [ 12%]
common/test/test_alert_data_store.py::test_get_data PASSED                                                            [ 25%]
common/test/test_datastore_backend.py::test_create_time_series_data_store PASSED                                      [ 37%]
common/test/test_datastore_backend.py::test_create_index PASSED                                                       [ 50%]
common/test/test_datastore_backend.py::test_save_data PASSED                                                          [ 62%]
common/test/test_datastore_backend.py::test_get_data PASSED                                                           [ 75%]
common/test/test_datastore_backend.py::test_delete_index PASSED                                                       [ 87%]
common/test/test_datastore_backend.py::test_delete_collection PASSED                                                  [100%]

===================================================== 8 passed in 7.00s =====================================================
```

### CODE REVIEW

File: const.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# See the GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+from enum import Enum
+
+
```
Comment (shaileshsvc) [inline const.py:22]: TODO: To move these to config file later

**Review by shaileshsvc — APPROVED** [review]

File: const.py (hunk 1 above)
Comment (akash2144) [reply const.py:22]: Done.

**Review by mukhtar-inamdar — APPROVED** [review]
Approved

**Comment by ajaysrivas** [comment]
Merging as it's already approved by @shaileshsvc and @mukhtar-inamdar.

**Pull request merged**